tests/Stormpath/Tests/Resource/AccountTest.php

```diff
@@ -361,8 +361,16 @@ public function testAddingCustomData()
         $customData = $account->customData;
         $this->assertEquals('unit Test', $customData->unitTest);
 
+        $customData = self::$account->customData;
+        $customData->locations = ['BuildingA', 'BuildingB'];
+        $customData->save();
 
```

Comment:
I would add an assertion here to make sure that the current value of `customData->locations` is `['BuildingA', 'BuildingB']`.